src/parser/filters/checkIntervalsConsistency.js

```diff
@@ -0,0 +1,28 @@
+import { hasAll } from '../../helpers/hasElement';
+
+const allForbiddenCombos = [
+	//['2',  '3'], // "2" modifier yields a ninth interval and not a second (eg, C2 = Cadd9)
+	['3',  'b3'],
+	//['3',  '4'], // valid in the Real Book: F#7SUS(add 3)
+	['5',  'b5'],
+	['5',  '#5'],
+	['b7', 'bb7'],
+	//['7',  'bb7'], // valid in the Real Book: C°7(add MA7)
```

Comment:
it definetely is a valid extension. 
This bb7 is often considered as a 6. For a more detailed explanation, diminished chords may contain any note in the diminished chord a tone above. So a `C°7` may contain any note in `D°7`.